docs/source/xgi-data.rst

```diff
@@ -30,7 +30,7 @@ See the `load_bigg_data() documentation <api/readwrite/xgi.readwrite.bigg_data.h
 Dataset format
 --------------
 
-The xgi-data format for higher-order datasets is a JSON data structure with the following structure:
+XGI-DATA provides higher-order datasets in two formats: (1) Hypergraph Interchange Format (HIF)-compliant JSON (2) XGI-specific JSON format. See the `HIF-standard <https://github.com/pszufe/HIF-standard>`_ documentation on format (1). Format (2) is structured as follows:
```

Comment:
Each dataset is available in only one of the two formats right?
Are they all (2) now and only new ones from now on will be (1)? 
Also, should we say that (1) is preferred in general from now on?

Reply:
There is a dataset formatted with (1) currently and I agree on adding that sentence to the documentation.

Reply:
Fixed.